splinterd/src/main.rs

```diff
@@ -68,30 +68,43 @@ const HEARTBEAT_DEFAULT: u64 = 30;
 
 #[cfg(not(feature = "config-toml"))]
 fn load_toml_config(config_file_path: &str) -> Config {
-    File::open(config_file_path)
-        .map_err(ConfigError::from)
-        .and_then(Config::from_file)
-        .unwrap_or_else(|err| {
-            warn!("Unable to load {}: {}", config_file_path, err);
-            Config::default()
-        })
+    match File::open(config_file_path) {
+        Ok(f) => Config::from_file(f),
+        Err(err) if err.kind() == io::ErrorKind::NotFound => {
+            debug!("Configuration file not found: {}", config_file_path);
```

Comment:
Should this still be warn?  I thought the concern about this message was the error message rather then the log level.

Reply:
@vaporos suggested that it be `debug` unless the config file was explicitly provided as a command-line arg (which we could do by moving the `unwrap_or` of the argument match into these methods).